hack/boilerplate.go.txt

```diff
@@ -13,3 +13,5 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
+// DO NOT EDIT.
```

Comment:
I am not 100% sure but I believe this could have some unintended consequences on non-generated files.

Reply:
I'll remove that commit since it's fixed in upstream. https://github.com/crossplaneio/crossplane/pull/704